easybuild/easyconfigs/r/rmats-turbo/rmats-turbo-4.1.1.patch

```diff
@@ -0,0 +1,12 @@
+diff -Nru rmats-turbo-4.1.1.orig/rMATS_C/Makefile rmats-turbo-4.1.1/rMATS_C/Makefile
+--- rmats-turbo-4.1.1.orig/rMATS_C/Makefile	2021-04-30 14:25:18.316325000 +0200
++++ rmats-turbo-4.1.1/rMATS_C/Makefile	2021-04-30 14:25:43.474876000 +0200
+@@ -10,7 +10,7 @@
+ endif
+ 
+ FFLAGS := -c
+-LDFLAGS := -lm -lgfortran -lgsl -lgslcblas -lgomp -lblas -llapack
++LDFLAGS := -lm -lgfortran -lgsl -lgslcblas -lgomp -lopenblas -lscalapack
```

Comment:
I wonder if we can instead use `$(LIBSCALAPACK)` environment variable here instead. This will be set to the correct link flags for any toolchain.